byway/src/main/webapp/destinations.js

```diff
@@ -234,5 +241,10 @@ else{
   initializeDestinationsPage();
 }
 
+/** Adds a tripId parameter to a URL string. Assumes the URL has no existing parameters. */
+function withParams(baseUrl, tripKey) {
```

Comment:
Update this function with a more descriptive name. "withParams" doesn't explain what the function is doing. A better name would be something like `buildUrlWithParams()` or `addParamsToUrl()`

Reply:
done